steps/redhat.py

```diff
@@ -166,3 +168,57 @@ def step_impl(context):
 	register_result =  context.remote_cmd(cmd='shell',
                     module_args='grep cc_rh_subscription.py /var/log/cloud-init.log | grep "not a valid key" | cut -d ":" -f4 | sed -e "s/^ //"')[0]['stdout']
 	assert register_result == 'list is not a valid key for rh_subscription. Valid keys are', "Error message not found"
+
+@when(u"import Red Hat's release key 2 to the superuser's keyring succeeds")
+def step_impl(context):
+    assert context.remote_cmd("command",
+           module_args="gpg --import /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"), "import of Red Hat's release key 2 failed"
+
+@then(u"verify if Red Hat's release key 2 matches public fingerprint")
+def step_impl(context):
+	gpg_out = context.remote_cmd("command", module_args="gpg --fingerprint")[0]['stdout']
+	m = re.search('567E 347A D004 4ADE 55BA  8A5F 199E 2F91 FD43 1D51',gpg_out)
+	assert m.group(0) == '567E 347A D004 4ADE 55BA  8A5F 199E 2F91 FD43 1D51', "the imported Red Hat's release key 2 does not match its public fingerprint"
+
+@when(u'download "{file}" script with sha256sum "{sha256sum}" finishes')
+def step_impl(context, file, sha256sum):
+	download_status = context.remote_cmd("get_url", module_args="url=" + file + " dest=/home/cloud-user/gpgverify.py force=yes sha256sum=" + sha256sum)
```

Comment:
Hardcoding `dest=/home/cloud-user/gpgverify.py` could cause problems if this code is ever reused.

Saving the script to `/tmp` should work.